ds4drv/__main__.py

```diff
@@ -151,6 +161,15 @@ def main():
         thread = create_controller_thread(index + 1, controller_options)
         threads.append(thread)
 
+    # Cleanup controller threads on SIGINT
+    def handle_sigint(signum, frame):
+        signal.signal(signal.SIGINT, signal.SIG_IGN)
+        cleanup_controller_threads(threads)
+        import sys
```

Comment:
Because I'm dumb. import sys; sys.exit(1) is a pretty common thing I use when debugging, and I didn't catch it before committing.